app-misc/anki/anki-24.06.3-r1.ebuild

```diff
@@ -757,22 +758,10 @@ GUI_RDEPEND="
 		dev-python/protobuf-python[${PYTHON_USEDEP}]
 		dev-python/requests[${PYTHON_USEDEP}]
 		dev-python/send2trash[${PYTHON_USEDEP}]
-		dev-python/waitress[${PYTHON_USEDEP}]')
-	qt6? (
-		dev-qt/qtsvg:6
-		$(python_gen_cond_dep '
-			>=dev-python/PyQt6-6.6.1[gui,network,opengl,quick,webchannel,widgets,${PYTHON_USEDEP}]
-			>=dev-python/PyQt6-sip-13.6.0[${PYTHON_USEDEP}]
-			>=dev-python/PyQt6-WebEngine-6.6.0[widgets,${PYTHON_USEDEP}]')
-	)
-	!qt6? (
-		dev-qt/qtgui:5[jpeg,png]
-		dev-qt/qtsvg:5
-		$(python_gen_cond_dep '
-			>=dev-python/PyQt5-5.15.5[gui,network,webchannel,widgets,${PYTHON_USEDEP}]
-			>=dev-python/PyQt5-sip-12.9.0[${PYTHON_USEDEP}]
-			>=dev-python/PyQtWebEngine-5.15.5[${PYTHON_USEDEP}]')
-	)
+		dev-python/waitress[${PYTHON_USEDEP}]
+		>=dev-python/PyQt6-6.6.1[gui,network,opengl,quick,webchannel,widgets,${PYTHON_USEDEP}]
+		>=dev-python/PyQt6-sip-13.6.0[${PYTHON_USEDEP}]
+		>=dev-python/PyQt6-WebEngine-6.6.0[widgets,${PYTHON_USEDEP}]')
```

Comment:
that block could use some better sorting after the merging from Qt separation into the main block.